### PR TITLE
[DO NOT MERGE][Feature][Checkout PRO] Wallet purpose

### PR DESCRIPTION
**Pull request by Kneehole**
## Description
En Checkout PRO se está incorporando una funcionalidad donde el integrador puede permitir únicamente pagos de usuarios logueados y medios online. Esta documentación que se agrega busca documentar esta nueva funcionalidad

### Checklist:
Before sending your contribution, please specify the following: 

- [x] This request modifies code snippets. 
- [x] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [x] Contribution includes new features. 
- [x] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/changelog/current-year.pt.md [hunk 1]
```diff
@@ -6,6 +6,20 @@ Descubra tudo sobre as novas versões e atualizações das integrações do Merc
 
 ## Agosto 2020
 
+### 27 de Agosto
```
Comment (pamalv7) [inline guides/changelog/current-year.pt.md:9]:
```suggestion
### 27 de agosto
```

File: guides/changelog/current-year.es.md [hunk 2]
```diff
@@ -6,6 +6,20 @@ Entérate todo sobre las nuevas versiones y actualizaciones de las integraciones
 
 ## Agosto 2020
 
+### 27 de Agosto
```
Comment (pamalv7) [inline guides/changelog/current-year.es.md:9]:
```suggestion
### 27 de agosto
```

File: guides/payments/web-payment-checkout/latest/integration.es.md [hunk 3]
```diff
@@ -224,15 +224,16 @@ curl -X POST \
 ```
 ]]]
 
+> NOTE
+>
+> Nota
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/integration.es.md:229]:
```suggestion
> Acepta pagos solo de usuarios registrados
```

File: guides/payments/web-payment-checkout/latest/integration.pt.md [hunk 4]
```diff
@@ -224,15 +224,17 @@ curl -X POST \
 ```
 ]]]
 
+> NOTE
+>
+> Nota
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/integration.pt.md:229]:
```suggestion
> Aceite pagamentos apenas de usuários cadastrados
```

File: guides/payments/web-payment-checkout/latest/integration.en.md [hunk 5]
```diff
@@ -224,15 +224,16 @@ curl -X POST \
 ```
 ]]]
 
+> NOTE
+>
+> Note
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/integration.en.md:229]:
```suggestion
> Accept payments only from registered users
```

File: guides/payments/web-payment-checkout/latest/integration.pt.md [hunk 6]
```diff
@@ -301,8 +303,8 @@ _Clique no link dentro do seu site e [teste a integração do seu Checkout Pro](
 
 > RIGHT_BUTTON_RECOMMENDED_PT
 >
-> Integração avançada
+> Aceite pagamentos apenas de usuários cadastrados
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/integration.pt.md:306]:
```suggestion
> Aceite somente usuários cadastrados
```

File: guides/payments/web-payment-checkout/latest/integration.pt.md [hunk 7]
```diff
@@ -224,15 +224,17 @@ curl -X POST \
 ```
 ]]]
 
+> NOTE
+>
+> Aceite pagamentos apenas de usuários cadastrados
```
Comment (Kneehole) [inline guides/payments/web-payment-checkout/latest/integration.pt.md:229]:
```suggestion
> Aceite somente usuários cadastrados
```

File: guides/payments/web-payment-checkout/latest/integration.pt.md [hunk 8]
```diff
@@ -224,15 +224,17 @@ curl -X POST \
 ```
 ]]]
 
+> NOTE
+>
+> Aceite somente usuários cadastrados
```
Comment (Kneehole) [inline guides/payments/web-payment-checkout/latest/integration.pt.md:229]:
```suggestion
> Aceite pagamentos apenas de usuários cadastrados
```

**Comment by pamalv7** [comment]
Cambios impactados en [PR #848](https://github.com/mercadopago/devsite-docs/pull/848/files) por nueva estructuración de files.

**Pull request closed without merging**